Designer/View/Components/PlacementSelectScreen.xaml

```diff
@@ -33,16 +33,16 @@
                 <ColumnDefinition Width="1*" />
                 <ColumnDefinition Width="1*" />
             </Grid.ColumnDefinitions>
-            <Button x:Name="RotateRightButton" Grid.Column="0" Background="Transparent" Height="40" Width="45" BorderThickness="0">
-                <Image Source="\Resources\Icons\rotate_right.png" Height="30" Width="30" />
+            <Button x:Name="RotateRightButton" Grid.Column="0" Background="Transparent" Height="40" Width="40" BorderThickness="0" Padding="0">
```

Comment:
Fixt dit de te grote afbeeldingen in de placement select screen?

Reply:
Nee, dit fixed dat de icons weer goed worden weergegeven en niet voor de helft. Dit werkte niet ivm material. icons kunnen kleiner als dat gewenst is

Reply:
Ah ja dat bedoelde ik, nice!